tianshou/data/batch.py

```diff
@@ -50,6 +50,12 @@ def _create_value(inst: Any, size: int) -> Union[
         return np.array([None for _ in range(size)])
 
 
+def _check_keys(keys):
```

Comment:
THis is nice, but are you sure it is the right PR to add this ?

Reply:
Also, I would clarify the name and call it `_assert_check`, since it is actually not checking but asserting.

Reply:
Please check pull reques #126 .